tools-v2/pkg/cli/command/curvebs/query/scanstatus/scanstatus.go

```diff
@@ -0,0 +1,156 @@
+/*
+*  Copyright (c) 2023 NetEase Inc.
+*
+*  Licensed under the Apache License, Version 2.0 (the "License");
+*  you may not use this file except in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+*  Unless required by applicable law or agreed to in writing, software
+*  distributed under the License is distributed on an "AS IS" BASIS,
+*  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*  See the License for the specific language governing permissions and
+*  limitations under the License.
+ */
+/*
+* Project: curve
+* Created Date: 2023-05-18
+* Author: lianzhanbiao
+ */
+
+package scanstatus
+
+import (
+	"fmt"
+	"log"
+
+	cmderror "github.com/opencurve/curve/tools-v2/internal/error"
+	cobrautil "github.com/opencurve/curve/tools-v2/internal/utils"
+	basecmd "github.com/opencurve/curve/tools-v2/pkg/cli/command"
+	"github.com/opencurve/curve/tools-v2/pkg/cli/command/curvebs/query/copyset"
+	"github.com/opencurve/curve/tools-v2/pkg/config"
+	"github.com/opencurve/curve/tools-v2/pkg/output"
+	"github.com/opencurve/curve/tools-v2/proto/proto/common"
+	"github.com/spf13/cobra"
+)
+
+const (
+	scanStatusExample = `$ curve bs query scan-satus --copysetid 1 --logicalpoolid 1`
+)
+
+type ScanStatusCommand struct {
+	CopysetInfoList []*common.CopysetInfo
+	basecmd.FinalCurveCmd
+}
+
+var _ basecmd.FinalCurveCmdFunc = (*ScanStatusCommand)(nil) // check interface
+
+func NewQueryScanStatusCommand() *ScanStatusCommand {
+	scanStatusCmd := &ScanStatusCommand{
+		FinalCurveCmd: basecmd.FinalCurveCmd{
+			Use:     "scan-status",
+			Short:   "quey ScanStatus Info in bs",
+			Example: scanStatusExample,
+		},
+	}
+
+	basecmd.NewFinalCurveCli(&scanStatusCmd.FinalCurveCmd, scanStatusCmd)
+	return scanStatusCmd
+}
+
+func NewScanStatusCommand() *cobra.Command {
+	return NewQueryScanStatusCommand().Cmd
+}
+
+func (cCmd *ScanStatusCommand) AddFlags() {
```

Comment:
maybe `sCmd` is better for `ScanStatusCommand `.

Reply:
done